robot/Cumulus/resources/NPSP.robot

```diff
@@ -594,4 +594,104 @@ Verify No Errors Displayed on AM Page And Object Group
     Page Should Not Contain Locator     gift_entry.page_error
     View Field Mappings Of The Object   Account 1
     Page Should Not Contain Locator     npsp_settings.page_error    warning     ${object_group} : ${field} (${field}__c)
-    Wait For Locator Is Not Visible     npsp_settings.field_error   ${field}__c
\ No newline at end of file
+    Wait For Locator Is Not Visible     npsp_settings.field_error   ${field}__c
+
+Change Object Permissions
+    [Documentation]  Adds or removes the Create, Read, Edit and Delete permissions for the specified object on the specified permission set..
+    [Arguments]  ${action}  ${objectapiname}  ${permset}
```

Comment:
Have to say. this is a really useful keyword.
Can you please add an example usage of the keyword here

Reply:
There is an example in Keywords.html on line 2618. I added examples in the doc for all new keywords.